src/Parser/DemonHunter/Havoc/CHANGELOG.js

```diff
@@ -4,6 +4,16 @@ import { Hewhosmites, Mamtooth } from 'MAINTAINERS';
 
```

Comment:
Missing imports for `SpellLink, SPELLS, ItemLink` and `ITEMS` causing Travis to fail.